proto/src/build.rs

```diff
@@ -1,10 +1,8 @@
-use std::io::Result;
-
-fn main() -> Result<()> {
-    let mut prost_build = prost_build::Config::new();
-    prost_build
+fn main() {
+    let file_descriptors = protox::compile(["proto/test_context.proto"], ["proto/"]).unwrap();
+    prost_build::Config::new()
         .type_attribute(".", "#[derive(serde::Serialize,serde::Deserialize)]")
         .extern_path(".google.protobuf.Timestamp", "::prost_wkt_types::Timestamp")
-        .compile_protos(&["proto/test_context.proto"], &["proto/"])?;
-    Ok(())
+        .compile_fds(file_descriptors)
+        .unwrap();
```

Comment:
Changing to protox was serendipitously done at the same time by Gabe. I changed this to be a little more succinctly/consistently by using `unwrap`s here—this is a `build.rs` file which is run prior to compilation, so `unwrap` is fine to use here